docroot/profiles/humsci/su_humsci_profile/src/HumsciContentAccessUpdater.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Drupal\su_humsci_profile;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\node\NodeInterface;
+
+/**
+ * Service for updating content access settings.
+ *
+ * The content access module allows different permissions per content type. The
+ * configuration (content_access.settings) is config ignored. This service will
+ * update relevent access setting per bundle or per node without affecting any
+ * existing settings.
+ */
+class HumsciContentAccessUpdater {
+
```

Comment:
Instead of `t()` function, should use the `StringTranslationTrait` or inject the service, then use `$this->t()`.

```suggestion
use StringTranslationTrait;
```